### DISCUSSION TITLE
Expression.Error: We cannot convert a value of type Record to type Text

### DISCUSSION THREAD

**Discussion by ckiotera** [tmijail/Odoo-Power-BI-Connector#28] [Q&A] [answered]
When I try to load data using the search_read method, I get the following error:

Expression.Error: We cannot convert a value of type Record to type Text.
Details:
    Value=[Record]
    Type=[Type]

Any ideas what I can do to get around this error? Thanks so much!

**Comment by tmijail** [comment]
I haven't seen that error before. 
Could you send me the complete code for the query that's throwing that error? (the code as shown in the [advanced editor](https://learn.microsoft.com/en-us/power-bi/transform-model/desktop-query-overview#advanced-editor))

**Reply by ckiotera** [reply]
Thanks for your reply! Here is the code:

`let
    Quelle = search_read("crm.lead", [fields={"id", "recurring_revenue", "probability"}])
in
    Quelle`

(Quelle is german for source)

**Reply by tmijail** [reply, marked as answer]
Try this:

```fs
let 
    Quelle = search_read("crm.lead", {}, [fields={"id", "recurring_revenue", "probability"}]) 
in 
    Quelle
```

Even though `search_domain` is optional, you need to specify it if you want to use any of the parameters that are to the right of it

**Reply by ckiotera** [reply]
Thank you so much, it works now!
